keras_hub/src/models/stable_diffusion_3/flow_match_euler_discrete_scheduler.py

```diff
@@ -27,7 +27,7 @@ class FlowMatchEulerDiscreteScheduler(layers.Layer):
     https://arxiv.org/abs/2403.03206).
     """
 
-    def __init__(self, num_train_timesteps=1000, shift=1.0, **kwargs):
+    def __init__(self, num_train_timesteps=1000, shift=3.0, **kwargs):
```

Comment:
I have found that we should use `3.0` here. The image quality is slightly improved and this value aligns with the settings in `huggingface/diffusers`.